docs/content/en/docs/getting-started/optional/irsa.md

```diff
@@ -101,9 +101,9 @@ You must use a single OIDC provider per EKS Anywhere cluster, which is the best
 
 	1. After the cluster is created you can grant service accounts access to the role by modifying the trust relationship. See the [How to use trust policies with IAM Roles](https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/) for more information on trust policies. Refer to [Configure the trust relationship for the OIDC provider's IAM Role](#configure-the-trust-relationship-for-the-oidc-providers-iam-role) for a working example.
 
-### Create the EKS Anywhere cluster
+### Create/upgrading the EKS Anywhere cluster
```

Comment:
```suggestion
### Create/upgrade the EKS Anywhere cluster
```

Reply:
Not sure why, but I changed this to `(or upgrade)` before committing. Locally I have no diff 🤷🏻 ... confused.